evap/static/scss/_adjustments.scss

```diff
@@ -140,6 +140,18 @@ $alert-colors: (
     white-space: normal;
 }
 
+.user-multi-select {
+    padding-right: 36px !important;
```

Comment:
@janno42 Should this be converted to another unit or is it fine like this? Note that there is one rule with `38px` a couple of lines above

Reply:
does the delete button have a fixed size or does it change when changing text sizes? if the button size is fixed, we can keep the pixel value.

Reply:
From what I found out zooming in and out in the browser, the width of the button doesn't really change (it grows and shrinks within an interval of less than 0.05px but not proportional to the browser's zoom). It's probably because the box containing all the user multi selects doesn't change its size with zooming in or out (at least in Chromium and Firefox).
Both the height and the padding-right are hard-coded in select2.min.css. 
So even if the outer box did change its size, I would expect the delete button to still be in the middle of the border of the outer box and the user multi selects.
Is that good enough? :sweat_smile:

Reply:
when changing the `font-size` of the document (changing the actual value, not zooming in the browser), the fixed width isn't correct anymore (see below). but as other things would break as well (e.g., the individual removal buttons) i don't mind keeping it like it is (and i hope we'll get rid of select2 at some point anyway).

![image](https://user-images.githubusercontent.com/1781719/148805351-2ef8c01c-bdf9-47c1-900a-3425f0df807e.png)